samples/apps/copilot-chat-app/webapp/src/libs/models/ChatMessage.ts

```diff
@@ -23,3 +23,5 @@ export interface ChatMessage {
     authorRole: AuthorRoles;
     debug?: string;
 }
+
+export type ChatMessages = IChatMessage[];
```

Comment:
I rarely see folks make type for the plural form (aka list/array) of an object. IMO, having the list/array inline (`IChatMessage[]`) is much clear than `ChatMessages`. I would love to learn the motivation behind this plural type.

Reply:
We use lists of IChatMessage[] way more often than just a single IChatMessage. I think having an alias can make things easier. I am open to reversing this change.

Reply:
I would appreciate it if we don't create the pattern of creating alias for plural type based on usage. Although type alias and the type doesn't impact developer exp, from readability perspective, it might be confusing.